itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -1527,17 +1538,33 @@ public void testDumpExternalTableSetTrue() throws Throwable {
             .verifyResult("t1")
             .run("show tables like 't2'")
             .verifyResult("t2")
+            .run("show tables like '" + extTabName1 + "'")
+            .verifyResult(extTabName1)
             .run("repl status " + replicatedDbName)
             .verifyResult(tuple.lastReplicationId)
             .run("select country from t2 where country = 'us'")
             .verifyResult("us")
             .run("select country from t2 where country = 'france'")
-            .verifyResult("france");
+            .verifyResult("france")
+            .run("select id from " + extTabName1)
+            .verifyResult("3")
+            .verifyExternalTable(replicatedDbName, "t1",
```

Comment:
test is required for partition location also , for both bootstrap and incremental load scenario

Reply:
Added a testcase for the case described above.